requirements.txt

```diff
@@ -1,141 +1,144 @@
 -i https://pypi.org/simple
-aiohttp==3.8.4 ; python_version >= '3.6'
+aiohttp==3.8.5 ; python_version >= '3.6'
 aiosignal==1.3.1 ; python_version >= '3.7'
-ape-solidity==0.6.8
+ape-etherscan==0.6.10
+ape-polygon==0.6.5
+ape-solidity==0.6.9
 appnope==0.1.3 ; sys_platform == 'darwin'
 asn1crypto==1.5.1
-asttokens==2.2.1
-async-timeout==4.0.2 ; python_version >= '3.6'
+asttokens==2.4.0
+async-timeout==4.0.3 ; python_version >= '3.7'
 attrs==23.1.0 ; python_version >= '3.7'
 backcall==0.2.0
 base58==1.0.3
-bitarray==2.7.3
-black==23.3.0
+bitarray==2.8.1
+black==23.9.1
 cached-property==1.5.2
-cachetools==5.3.0 ; python_version ~= '3.7'
-certifi==2023.5.7 ; python_version >= '3.6'
+cachetools==5.3.1 ; python_version >= '3.7'
+certifi==2023.7.22 ; python_version >= '3.6'
 cffi==1.15.1
-cfgv==3.3.1 ; python_full_version >= '3.6.1'
-chardet==5.1.0 ; python_version >= '3.7'
-charset-normalizer==3.1.0 ; python_full_version >= '3.7.0'
-click==8.1.6 ; python_version >= '3.7'
+cfgv==3.4.0 ; python_version >= '3.8'
+chardet==5.2.0 ; python_version >= '3.7'
+charset-normalizer==3.2.0 ; python_full_version >= '3.7.0'
+click==8.1.7 ; python_version >= '3.7'
 coincurve==18.0.0
 colorama==0.4.6 ; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4, 3.5, 3.6'
 commonmark==0.9.1
-cryptography==40.0.2
-cytoolz==0.12.1 ; implementation_name == 'cpython'
+cryptography==41.0.3
+cytoolz==0.12.2 ; implementation_name == 'cpython'
 dataclassy==0.11.1 ; python_version >= '3.6'
 decorator==5.1.1 ; python_version >= '3.5'
-deprecated==1.2.13 ; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
-distlib==0.3.6
+deprecated==1.2.14 ; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3'
+distlib==0.3.7
 eip712==0.2.1 ; python_version >= '3.8' and python_version < '4'
-eth-abi==4.1.0 ; python_version >= '3.7' and python_version < '4'
+eth-abi==4.2.1 ; python_full_version >= '3.7.2' and python_version < '4'
 eth-account==0.8.0 ; python_version >= '3.6' and python_version < '4'
 eth-ape==0.6.19
```

Comment:
In the scripts below you mention you  use 0.6.20. Maybe not super relevant, but it may be the case that something in this PR was version-specific.